frontend/src/routes/Identities/Users/UserDetails.tsx

```diff
@@ -0,0 +1,73 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { PageSection, Stack, Text } from '@patternfly/react-core'
+import { DescriptionList, DescriptionListTerm, DescriptionListGroup, DescriptionListDescription } from '@patternfly/react-core'
+import { useParams } from 'react-router-dom-v5-compat'
+import { useTranslation } from '../../../lib/acm-i18next'
+import { useQuery } from '../../../lib/useQuery'
+import { listUsers } from '../../../resources/rbac'
+import { useMemo } from 'react'
+import AcmTimestamp from '../../../lib/AcmTimestamp'
+import { AcmLoadingPage } from '../../../ui-components'
+
+const UserDetails = () => {
```

Comment:
either `UserDetails`,  `UserGroups`,  `UsersRoleAssigments` should be information retrieval agnostic, meaning they should be wrapped up by upper level component (what about to use `UserDetail` (aka `UserDetailPage`) for it? ) and then to expect `user: User` as a property?

Reply:
I miss "restricted" use case here, like the identities route should be rbaced, right? and then to redirect to `/`, right?